puzlib/src/main/java/app/crossword/yourealwaysbe/puz/Playboard.java

```diff
@@ -732,7 +766,30 @@ public Word playLetter(char letter) {
             return null;
         }
 
-        if (preserveCorrectLettersInShowErrors && b.getResponse() == b.getSolution() && isShowErrors()) {
+        if (this.scratchMode) {
+            pushNotificationDisabled();
+
+            Note note = this.getNote();
+            String response = this.getCurrentWordResponse();
+
+            // Create a note for this clue if we don't already have one
+            if (note == null) {
+                note = new Note(response);
```

Comment:
What's the rationale here? -- it creates a note with the free text populated by the current board contents. Potentially this is a sequence of BLANKs (spaces) only. Why prefill the free text at all?

Reply:
Aha -- i think i see. This was supposed to copy the current board to the scratch, else, when you fill in the scratch from the playactivity there are gaps where the main response is. Perhaps once this is fixed to copy the response into the scratch, there's no need to create a blank string to prefix the scratch field with?

Reply:
I'd actually got confused!  I thought the `text` field held the main answer from the board to populate the miniboard in the NotesActivity - I hadn't realised that was populated directly from the Playboard.

I've fixed this now.  I'm still a bit uncomfortable about the fields in Note being initialised to nulls, but I've done a bunch of manual testing and it seems to be okay.